draft-ietf-quic-transport.md

```diff
@@ -5595,6 +5595,14 @@ frame multiple times MUST NOT be treated as a connection error.  A receiver can
 use the sequence number supplied in the NEW_CONNECTION_ID frame to identify new
 connection IDs from old ones.
 
+When receiving a NEW_CONNECTION_ID frame carrying a sequence number greater than
+those that have been previously received, an endpoint MUST check that the sum of
```

Comment:
Something like this seems like a sensible check I thought we already had?  If not, adding it seems separable from this PR?

Reply:
IIUC, we have a MUST on the issuer-side, but nothing on the consumer-side. Therefore it is my understanding is that the status quo is equivalent to a consumer MAY check and call it a protocol violation.

I think status quo is fine; MAY check is fine when the consumer of the CIDs track each of them individually.

However, in this PR, a RCID frame is expected to carry all the CIDs being retired from the beginning of the connection, and the gaps to be no more than the `max_connection_id_limit`. To meet that expectation, there has to be a check that detects the issuer intentionally creating gaps.

That's why a MUST is added here.